ou_dedetai/config.py

```diff
@@ -59,55 +60,55 @@
     globals()[key] = os.getenv(key, default)
 
 # Set other run-time variables not set in the env.
-ACTION = 'app'
-APPIMAGE_FILE_PATH = None
-authenticated = False
-BADPACKAGES = None
+ACTION: str = 'app'
+APPIMAGE_FILE_PATH: Optional[str] = None
+authenticated = False # FIXME: no references
+BADPACKAGES: Optional[str] = None # FIXME: no references
```

Comment:
This is leftover from when we were still using this for removing AppImageLauncher. We ended up having to handle AppImageLauncher differently, but we kept this in case we ever discovered a problematic package.